JavaScript/packages/recognizers-date-time/src/dateTime/chinese/datePeriodConfiguration.ts

```diff
@@ -713,6 +725,86 @@ export class ChineseDatePeriodParser extends BaseDatePeriodParser {
         return this.parseCommonDurationWithUnit(beforeStr, sourceUnit, numStr, referenceDate);
     }
 
+    protected parseDecade(source: string, referenceDate: Date): DateTimeResolutionResult {
+        let result = new DateTimeResolutionResult();
+        let century = Math.floor(referenceDate.getFullYear() / 100) + 1;
+        let decadeLastYear = 10;
+        let inputCentury = false;
+        let beginLuisStr;
+        let endLuisStr;
+
+        let match = RegExpUtility.getMatches(this.decadeRegex, source).pop();
+        if (!match) {
+            return result;
+        }
+
+        let decadeStr = match.groups('decade').value;
+        let decade = this.convertChineseToNumber(decadeStr);
+        let centuryStr = match.groups('century').value;
+        if (centuryStr){
```

Comment:
Nitpick: Please follow the code standard consistently. The are some missing whitespaces before opening the curly brakcets. Same in line 748 and elsewhere. Some other issues further down too.